test/functionalTest/cases/0876_entity_dates/filter_datecreated_sub.test

```diff
@@ -42,23 +42,23 @@ accumulatorStart
 
 T_0=$(TZ=GMT date +"%Y-%m-%dT%H:%M:%S.00Z")
 T_2=$(TZ=GMT date +"%Y-%m-%dT%H:%M:%S.00Z" --date="2seconds")
-T_4=$(TZ=GMT date +"%Y-%m-%dT%H:%M:%S.00Z" --date="4seconds")
+T_6=$(TZ=GMT date +"%Y-%m-%dT%H:%M:%S.00Z" --date="6seconds")
 
-echo "01. At t_0: Create subscription with q: dateCreated==t_2..t_4"
+echo "01. At t_0: Create subscription with q: dateCreated==t_2..t_6"
 echo "============================================================="
 payload='
 {
     "subject": {
         "entities": [
             {
-                "idPaatern": "E.*",
```

Comment:
idPaatern? How this was working before the change? Maybe because fields different from id or idPattern is interpreted as "no field" and, in sequence, "no field" is intepreted as ".*" pattern?

This case passed unnoticed until now supports the (already taken decission) of being explicit regarding id/idPattern and don't allow implicit defaults ;)

NTC